registry/manifests/kabanero-operator/0.6.0/kabanero-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -178,6 +178,15 @@ spec:
         - kind: Task
           name: ''
           version: v1alpha1
+        - kind: Condition
+          name: ''
+          version: v1alpha1
+        - kind: TriggerBinding
```

Comment:
The trigger bindings and trigger templates don't follow the object ownership model (they are in a different namespace) so there will never be any of these objects in the openshift console.  It's probably best to leave them off of the resources view (unless there's another reason why they are here, in which case please let me know what that is).